doc/connectivity/bluetooth/api/mesh/srpl_cli.rst

```diff
@@ -3,7 +3,46 @@
 Solicitation PDU RPL Configuration Client
 #########################################
 
-Solicitation PDU RPL Configuration Client model is used for clearing the solicitation replay protection list (SRPL) of a node that supports the Solicitation PDU RPL Configuration Server model.
+The Solicitation PDU RPL Configuration Client model (SRPL Config Client) support the functionality of removing addresses from the solicitation
+replay protection list (SRPL) of a node that supports the :ref:`bluetooth_mesh_srpl_srv` model.
+
+The SRPL Config Client is an optional mesh subsystem that can be enabled through the :kconfig:option:`BT_MESH_SOL_PDU_RPL_CLI` option.
+
+Remove entries from target SRPL
+*******************************
+
+The SRPL Config Client can clear entries in a target by using the :c:func:`bt_mesh_sol_pdu_rpl_clear` call:
+
+.. code-block:: C
+
+      uint16_t start_rsp;
+      uint8_t len_rsp;
+
+      const struct bt_mesh_msg_ctx my_ctx = {
+         .addr = 0x0004,
+         .net_idx = 0,
+         .send_ttl = BT_MESH_TTL_DEFAULT,
+      };
+
+      uint16_t range_start = 5;
+      uint8_t range_len = 20;
+
+      bt_mesh_sol_pdu_rpl_clear(&my_ctx, range_start, range_len,
+                                &start_rsp, &len_rsp);
```

Comment:
Same as for On-Demand Private Proxy Client model. I'm not sure if we need such level of details. A user should be able to perceive this from the API description. If we have bad API description, we should fix it there.